Documentation/DE/Referenz/Hooks/Index.rst

```diff
@@ -16,115 +16,318 @@
 Hooks
 ^^^^^
 
+.. warning::
+    Um Hooks nutzen zu können, benötigen Sie tiefgehende Kenntnisse über PHP-Klassen,
+    die Implementation von Interfaces und der seminars Objekt Interna.
 
-New hooks for the single view
-"""""""""""""""""""""""""""""
+Mit Hilfe von Hooks erweitern Sie die Funktionalität von seminars ohne XCLASSen zu nutzen.
+Es gibt Hooks in diese Teile von seminars:
 
-There now are two new hooks for the single view. They are registered
-like this in ext\_localconf.php:
+* :ref:`singleview`
+* :ref:`listview`
+* :ref:`registrationform`
+* :ref:`notificationemail`
+* :ref:`emailsalutation`
+* :ref:`backendemail`
 
-::
+Bitte nehmen Sie Kontakt zu uns auf, wenn Sie weitere Hooks benötigen.
 
-   $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['seminars']['singleView'][]
-         = \Tx_Seminarspaypal_Hooks_EventSingleView::class;
+.. important::
+    seminars wird derzeit grundlegend überarbeitet, um es an die Weiterentwicklung von TYPO3
+    Programmiertechniken anzupassen. Wir bemühen uns, die Änderungen so gering wie möglich zu
+    halten. Informieren Sie sich über Änderungen in CHANGELOG.md, den DocBlocks der Interfaces
+    die Sie implementieren und in diesem Kapitel der Dokumentation, bevor Sie auf eine neue
+    Haput-Version updaten.
 
-They are used like this:
+.. _singleview:
 
-::
+Hooks zur Einzelansicht
+"""""""""""""""""""""""
 
-   class Tx_Seminarspaypal_Hooks_SingleView implements Tx_Seminars_Interface_Hook_EventSingleView {
+.. important::
+    Die Nutzung von :php:`\Tx_Seminars_Interface_Hook_EventSingleView` ist veraltet seit
+    seminars 3. Es wird in seminars 4 entfernt werden. Bitte aktualisieren Sie auf
+    :php:`\OliverKlee\Seminars\Hooks\Interfaces\SeminarSingleView`.
 
-/\*\*
+Es gibt einen Hook in die Einzelansicht. Er wird aufgerufen, bevor das Template zu HTML
+umgewandelt wird. Sie können damit eigene Marker ausfüllen oder bestehende Marker-Werte 
+verändern. Für Details zu Eigenschaften und Methoden siehe :file:`Classes/Frontend/DefaultController.php`.
 
-\* Modifies the event single view.
+Ihre Klasse, die :php:`\OliverKlee\Seminars\Hooks\Interfaces\SeminarSingleView` implementiert,
+machen Sie seminars in :file:`ext_localconf.php` Ihrer Extension bekannt:: php
```

Comment:
Please use `.. code-block:: php` here (like this: https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CheatSheet.html#code-block-directive).